aiida/plugins/factories.py

```diff
@@ -248,6 +248,29 @@ def SchedulerFactory(entry_point_name: str, load: bool = True) -> Optional[Union
     raise_invalid_type_error(entry_point_name, entry_point_group, valid_classes)
 
 
+def StorageFactory(entry_point_name: str, load: bool = True) -> Optional[Union[EntryPoint, 'StorageBackend']]:
```

Comment:
We need to add some tests for this in the `tests.plugins.test_factories.py`, right?

(btw I just noticed there are no tests for the `GroupFactory` either. Not saying to add them in this PR though, hehe, just mentioning)

Reply:
Added the tests for the `StorageFactory`